planning/behavior_path_planner/autoware_behavior_path_static_obstacle_avoidance_module/include/autoware/behavior_path_static_obstacle_avoidance_module/utils.hpp

```diff
@@ -120,6 +206,12 @@ void fillObjectStoppableJudge(
 
 void updateClipObject(ObjectDataArray & clip_objects, AvoidancePlanningData & data);
 
+/**
+ * @brief compensate lost objects until a certain time elapses.
```

Comment:
Explanation for `ObjectDataArray & stored_objects` is missing.
Should be something like this, I guess.
```suggestion
 * @brief compensate lost objects until a certain time elapses.
 * @param previous stopped object
```